lib/galaxy/managers/hdcas.py

```diff
@@ -42,6 +42,27 @@ def __init__( self, app ):
         """
         super( HDCAManager, self ).__init__( app )
 
+    def map_datasets( self, content, fn, *parents ):
```

Comment:
I'm not trying to nit pick - I intend to merge this - but I did want to try to reach some consensus with you about managers here. I would argue that this belongs on the model instead of in a manager - my perception of managers is that if it  involves ``app`` or ``trans`` it belongs in a manager. If the operation is self contained like this and the model can do everything itself or with its associations - it belongs in the model.